DMX.Gatekeeper.Api/Brokers/DmxApis/IDmxApiBroker.Commands.cs

```diff
@@ -0,0 +1,14 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System.Threading.Tasks;
+using DMX.Gatekeeper.Api.Models.Commands;
+
+namespace DMX.Gatekeeper.Api.Brokers.DmxApis
+{
+    public partial interface IDmxApiBroker
+    {
+        ValueTask PostCommand(LabCommand labCommand);
```

Comment:
We should always return what we post. especially at the foundation levels (brokers or foundation services)